test/manual/kerberos.test.js

```diff
@@ -0,0 +1,352 @@
+'use strict';
+
+const format = require('util').format;
+const test = require('../functional/shared').assert;
+const { MongoClient } = require('../../src');
+
+// You need to set up the kinit tab first
+// https://wiki.mongodb.com/pages/viewpage.action?title=Testing+Kerberos&spaceKey=DH
+// kinit -p drivers@LDAPTEST.10GEN.CC
+// password: (not shown)
+
+describe('Kerberos', function () {
+  if (process.env.MONGODB_URI == null) {
+    console.log('skipping Kerberos tests, MONGODB_URI environment variable is not defined');
+    return;
+  }
+  if (process.platform === 'win32') {
+    console.log('Only running Win32 tests');
+    return;
+  }
+
+  it.only('Should Correctly Authenticate using kerberos with MongoClient', function (done) {
```

Comment:
there's an `it.only` left here. Also, nit: `'should authenticate using kerberos`

Reply:
Just to clear up what's going on here, I added `.only` to the two tests that I fixed from the original disabled test file. I haven't touched the other tests, which either don't run because they are win32-specific and we don't currently test windows, or because they don't have `.only`.